### PR TITLE
Libtorrent

### PR DESCRIPTION
**Pull request by pauldreik**
Hi,
I have discussed bringing [libtorrent](https://www.libtorrent.org/) to oss-fuzz with Arvid, the libtorrent author, and he was positive. This branch builds fine locally with the commands suggested on [new project guide](https://github.com/google/oss-fuzz/blob/master/docs/new_project_guide.md).
For now, there are three fuzz targets but if things work fine I will add more.
Thanks,
Paul

### CODE REVIEW

**Comment by pauldreik** [comment]
Ah sorry for accidentally bringing in an unrelated boost commit which has already been merged. I just looked at the diff, not the commits. My bad.

File: projects/libtorrent/build.sh [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+# build project
+# e.g.
```
Comment (Dor1s) [inline projects/libtorrent/build.sh]: please remove these default lines which are commented out

Comment (pauldreik) [reply projects/libtorrent/build.sh]: Done.

**Comment by inferno-chromium** [comment]
@mbarbella-chromium - sheriffing ping for review.

**Pull request merged**